integrations/openzeppelin/contracts.md

```diff
@@ -0,0 +1,235 @@
+ - [**ERC1155**](https://docs.openzeppelin.com/contracts/4.x/erc1155) — also known as the multi-token contract, because it can represent both fungible and non-fungible tokens in a single smart contract. It follows [EIP-1155](https://eips.ethereum.org/EIPS/eip-1155)
+
+The wizard is comprised of the following sections:
+
+ 1. **Token standard selection** — shows all the different standards supported by the wizard
+ 2. **Settings** — provides the baseline settings for each token standard, such as token name, symbol, pre-mint (token supply when the contract is deployed), and URI (for non-fungible tokens)
+ 3. **Features** — list of all features available for each token standard. You can find more information about the different features in the following links:
+     - [ERC20](https://docs.openzeppelin.com/contracts/4.x/api/token/erc20)
+     - [ERC721](https://docs.openzeppelin.com/contracts/4.x/api/token/erc721)
+     - [ERC1155](https://docs.openzeppelin.com/contracts/4.x/api/token/erc1155)
+ 4. **Access Control** — list all of the available [access control mechanism](https://docs.openzeppelin.com/contracts/4.x/access-control) available for each token standard
+ 5. **Interactive code display** — shows the smart contract code with the configuration as set by the user
+
+![OpenZeppelin Contracts Wizard](/images/openzeppelin/ozwizard-images1.png)
+
+Once you have set up your contract with all the settings and features, it is just as easy as copying and pasting the code into your contract file.
+
+## Deploying OpenZeppelin Contracts on Moonbeam
+
+This section goes through the steps for deploying OpenZeppelin contracts on Moonbeam Currently. It covers the following contracts:
+
+ - ERC20 (fungible tokens)
+ - ERC721 (non-fungible tokens)
+ - ERC1155 (multi-token standard)
+
+All the code of the contracts was obtained using OpenZeppelin [Contract Wizard](https://docs.openzeppelin.com/contracts/4.x/wizard).
+ 
+### Checking Prerequisites
+
+The steps described in this section assume you have [MetaMask](https://metamask.io/) installed and connected to the Moonbase Alpha TestNet. Contract deployment is done using the [Remix IDE](https://remix.ethereum.org/) via the "Injected Web3" environment. You can find corresponding tutorials in the following links:
+
+ - [Interacting with Moonbeam using MetaMask](/integrations/wallets/metamask/)
+ - [Interacting with Moonbeam using Remix](/integrations/remix/)
+
+### Deploying an ERC20 Token
+
+For this example, an ERC20 token will be deployed to Moonbase Alpha. The final code used combines different contracts from OpenZeppelin:
+
+ - **ERC20.sol** — ERC20 token implementation with the optional features from the base interface. Includes the supply mechanism with a `mint` function but needs to be explicitly called from within the main contract
```

Comment:
Is this line referring to the "Features" section of the Contract Wizard? Also not sure what you mean by "the supply mechanism" 
<img width="261" alt="Screen Shot 2021-05-20 at 4 35 15 PM" src="https://user-images.githubusercontent.com/26533957/119057258-61f47180-b989-11eb-8c93-3e5b2b0e956d.png">

Reply:
If you check when you click Mintable nothing really changes on the contract template, it just adds the Ownable Library. The mint function is included in the `ERC20.sol` by default in OpenZeppelin, even though is not a standard function of the ERC20 interface.

Supply mechanism in this part means how are tokens created, usually via minting